src/groupeddataframe/splitapplycombine.jl

```diff
@@ -235,21 +220,7 @@ julia> df = DataFrame(a = repeat([1, 2, 3, 4], outer=[2]),
 
 julia> gd = groupby(df, :a);
 
-julia> map(:c => sum, gd)
```

Comment:
Yes - I will revert it. But I have a question here - what is the upper limit you think is reasonable for the length of docstrings?

Reply:
No idea. I guess we shouldn't bother too much about that -- ideally the UI/REPL would show the top of the docstring and allow scrolling. It would be too bad to avoid documenting things for practical reasons.